.github/workflows/ci.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
             fail-fast: false
             matrix:
                 php: [ '7.2', '7.3', '7.4', '8.0', '8.1' ]
-                monolog: [ '1.*', '2.*' ]
+                monolog: [ '1.*', '2.*', '3.*@dev' ]
```

Comment:
This matrix spawns jobs with PHP 7.2 and Monolog 3 which is pointless.

Instead, let's add an entry to the `include` section:

```YAML
-   php: '8.1'
    monolog: '3.x@dev'
```

Reply:
Yeah.. this was a rush job and it shows 😅 Thanks